### ISSUE TITLE
Crash when joining high Crime Spree lobbies

### ISSUE THREAD

**Issue by TheHellmare** [KashEight/CustomDiscordStatus#1]
Mod causes the game to crash when joining a high Crime Spree lobby

Application has crashed: C++ exception
mods/CustomDiscordStatus/lua/PlatformManager.lua:238: attempt to index a nil value

SCRIPT STACK

apply_status() @mods/CustomDiscordStatus/lua/PlatformManager.lua:189
set_rich_presence_discord() @mods/CustomDiscordStatus/lua/PlatformManager.lua:59
set_rich_presence() lib/managers/platformmanager.lua:385
at_exit() lib/states/ingamewaitingforplayers.lua:619
trans_func() lib/states/gamestate.lua:14
_do_state_change() core/lib/utils/game_state_machine/coregamestatemachine.lua:124
end_update() core/lib/utils/game_state_machine/coregamestatemachine.lua:104
end_update() lib/setups/setup.lua:891
end_update() lib/setups/gamesetup.lua:775
end_update() lib/setups/networkgamesetup.lua:33
core/lib/setups/coresetup.lua:583

-------------------------------

Callstack:

         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     zip_get_name                                        
         payday2_win32_release  (???)     zip_get_name                                        

-------------------------------

Current thread: Main

**Comment by KashEight** [comment]
I tested, and checked this error happened under this situation: 
- Joining high ping server (for example, connect between Asia and USA)

This error did not happen when low ping server, I tested under the environment: the server my steam friend living in the same country I live hosted.

I want to fix not to cause this error, so will keep this issue.
It will take much times to fix, and sorry for about it.
